numpy/typing/mypy_plugin.py

```diff
@@ -61,13 +61,23 @@ def _get_extended_precision_list() -> t.List[str]:
     return [i.__name__ for i in extended_types if i.__name__ in extended_names]
 
 
+def _get_c_intp_name() -> str:
+    if np.ctypeslib.c_intp is np.intp:
+        return "c_int64"  # Plan B, in case `ctypes` fails to import
```

Comment:
Is this safe?

Reply:
During runtime? Yes, if `ctypes` fails to load then `np.ctypeslib.c_intp` will default to `np.intp` such that aforementioned variable can always be imported.
Of course we can't guarantee that `ctypes.c_int64` is actually the correct type in such case, 
but based on the prevalence of 64-bit systems these days it most likely is (the alternative being returning `Any`).

https://github.com/numpy/numpy/blob/5520a1757e4476bc6526973ad46c46b82637ae1c/numpy/ctypeslib.py#L66-L85

Reply:
Would something like
```
In [3]: np.intp is np.int64                                                     
Out[3]: True
```
work?

Reply:
Or maybe
```
In [7]: np.iinfo(np.intp).dtype                                                 
Out[7]: dtype('int64')
```